backend/graphing_data.py

```diff
@@ -0,0 +1,18 @@
+import pandas as pd
+
+
+# This is the GraphingData class.
+# Only a DataFrame with two columns can be store here.
+# Column 1 must be labelled "date" and column 2 must be
+# labelled either "frequency" or "revenue".
+class GraphingData:
+    data: pd.DataFrame
+
+    def __init__(self, data: pd.DataFrame):
+        assert data.shape[1] == 2, "Data must have exactly 2 columns."
+        assert data.columns[0] == "data", "First column header must be 'data'."
```

Comment:
make sure that is a datetime.date object, and column name should be "date" not data